corehq/apps/users/migrations/0046_add_default_mobile_worker_role.py

```diff
@@ -0,0 +1,35 @@
+# Adds the default mobile worker role to existing projects that don't yet have one.
+from django.db import migrations
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.users.models import Permissions
+from corehq.apps.users.models_role import UserRole
+from corehq.apps.users.role_utils import UserRolePresets
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _add_default_mobile_worker_role(apps, schema_editor):
+    default_mobile_worker_roles = UserRole.objects.filter(is_commcare_user_default=True)
```

Comment:
nit: you can save some queries by doing something like this instead:
```suggestion
    domains_with_default = set(UserRole.objects.filter(is_commcare_user_default=True).values_list('domain', flat=True))
```
as written, it'll perform a `UserRole` query per domain

Reply:
> as written, it'll perform a UserRole query per domain

Do you know why? I'd have assumed the existing code becomes a single `select * from users_userrole where is_commcare_user_default = true`

(This is for my own edification - django ORM performance is **killing** one of my personal projects lately.)

Reply:
Querysets are evaluated lazily, and only once, but the `.filter` inside the loop makes a new queryset each time - that modifies the query itself, not the results returned.  You could also perform one big query up front, but you'd need to filter the results in memory, not using the ORM.  I was able to verify this locally by playing with a queryset using
```./manage.py shell_plus --print-sql```